types/es-abstract/5/StrictEqualityComparison.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function StrictEqualityComparison(x: unknown, y: unknown): boolean;
```

Comment:
can this be a predicate that asserts that x is the same type as y?

Reply:
**TypeScript** doesn’t currently provide a way to declare a function as being equivalent to `x === y`.

Reply:
not `===`, but `x is typeof y`?